### PR TITLE
docs: provide instructions on getting enterprise file from new license Teleport Account

### PR DESCRIPTION
**Pull request by stevenGravy**
Replaces dashboard.gravitational.com to use new dedicated Teleport Account.

### CODE REVIEW

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 1]
```diff
@@ -25,13 +25,12 @@ government agencies.
...
 | [SC-12 Cryptographic Key Establish and Management]((=fedramp.control_url=)SC-12) | Teleport initializes cryptographic keys that act as a Certificate Authority (CA) to further issue x509 and SSH certificates. SSH and x509 user certificates that are issued are signed by the CA and are (by default) short-lived. SSH host certificates are also signed by the CA and rotated automatically (a manual force rotation can also be performed).<br/>Teleport Enterprise builds against a FIPS 140-2 compliant library (BoringCrypto) is available. <br/>In addition, when Teleport Enterprise is in FedRAMP/FIPS 140-2 mode, Teleport will only start and use FIPS 140-2 compliant cryptography. |
 
-Enterprise customers can download the custom FIPS package from the [Dashboard](https://dashboard.gravitational.com/web/login).  Look for `Linux 64-bit (FedRAMP/FIPS)`. RPM and DEB packages are also available.
-
 ## Setup
 
-Visit the [Teleport Customer
+Visit your [Teleport Account](https://teleport.sh) or the legacy [Teleport Customer
```
Comment (klizhentas) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:30]: We will deprecate `dashboard.gravitational.com` soon. There is no need to keep any references. Any customer who needs new license, can get it from `teleport.sh` instead.

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 2]
```diff
@@ -64,7 +64,9 @@ See the [SSO](../../access-controls/sso.mdx) chapter for more details.
 
 With Teleport we have built the foundation to meet FedRAMP requirements for the purposes of accessing infrastructure. This includes support for [FIPS 140-2](https://en.wikipedia.org/wiki/FIPS\_140-2), also known as the Federal Information Processing Standard, which is the US government approved standard for cryptographic modules.
 
-Enterprise customers can download the custom FIPS package from the [Gravitational Dashboard](https://dashboard.gravitational.com/web/).
+Enterprise customers can download the custom FIPS package from their [Teleport Account](https://teleport.sh) or
```
Comment (klizhentas) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:67]: Same here

**Review by klizhentas — CHANGES_REQUESTED** [review]

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/access-controls/compliance-frameworks/fedramp.mdx]: done

Comment (stevenGravy) [reply docs/pages/access-controls/compliance-frameworks/fedramp.mdx]: Please take a look when you can @klizhentas

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx (hunk 2 above)
Comment (stevenGravy) [reply docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx]: This was done across all files.

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 3]
```diff
@@ -25,13 +25,11 @@ government agencies.
...
 | [SC-12 Cryptographic Key Establish and Management]((=fedramp.control_url=)SC-12) | Teleport initializes cryptographic keys that act as a Certificate Authority (CA) to further issue x509 and SSH certificates. SSH and x509 user certificates that are issued are signed by the CA and are (by default) short-lived. SSH host certificates are also signed by the CA and rotated automatically (a manual force rotation can also be performed).<br/>Teleport Enterprise builds against a FIPS 140-2 compliant library (BoringCrypto) is available. <br/>In addition, when Teleport Enterprise is in FedRAMP/FIPS 140-2 mode, Teleport will only start and use FIPS 140-2 compliant cryptography. |
 
-Enterprise customers can download the custom FIPS package from the [Dashboard](https://dashboard.gravitational.com/web/login).  Look for `Linux 64-bit (FedRAMP/FIPS)`. RPM and DEB packages are also available.
-
 ## Setup
 
-Visit the [Teleport Customer
-Portal](https://dashboard.gravitational.com/web/login) to download a FIPS binary
-for Teleport Enterprise.
+Visit your [Teleport Account](https://teleport.sh) to download a FIPS binary
```
Comment (zmb3) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:30]:
```suggestion
Sign in to your [Teleport Account](https://teleport.sh) to download a FIPS binary
```

File: docs/pages/application-access/guides/connecting-apps.mdx [hunk 4]
```diff
@@ -103,7 +103,7 @@ e.g. `grafana.teleport.example.com`. You can also override `public_addr` e.g
 `grafana.acme.com` if you configure the appropriate DNS entry to point to the
 Teleport proxy server.
 
-## Start application service with a config file
+## Start Teleport Application Service with a config file
```
Comment (ptgott) [inline docs/pages/application-access/guides/connecting-apps.mdx:106]:
```suggestion
## Start the Teleport Application Service with a config file
```

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 5]
```diff
@@ -64,7 +64,8 @@ See the [SSO](../../access-controls/sso.mdx) chapter for more details.
 
 With Teleport we have built the foundation to meet FedRAMP requirements for the purposes of accessing infrastructure. This includes support for [FIPS 140-2](https://en.wikipedia.org/wiki/FIPS\_140-2), also known as the Federal Information Processing Standard, which is the US government approved standard for cryptographic modules.
 
-Enterprise customers can download the custom FIPS package from the [Gravitational Dashboard](https://dashboard.gravitational.com/web/).
+Enterprise customers can download the custom FIPS package from their [Teleport Account](https://teleport.sh).
```
Comment (ptgott) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:67]:
```suggestion
Enterprise customers can download the custom FIPS package from their [Teleport account](https://teleport.sh).
```
"Account" shouldn't be capitalized here

File: docs/pages/choose-an-edition/teleport-enterprise/license.mdx [hunk 6]
```diff
@@ -17,10 +17,13 @@ your license file.
 
 ### Download your license file
 
-To obtain your license file, visit the [Teleport customer
-portal](https://dashboard.gravitational.com/web/login) and log in. Click
-"DOWNLOAD LICENSE KEY". You will see your current Teleport Enterprise account
-permissions and the option to download your license file:
+Teleport provides a dedicated Teleport Account site to download your license,
+enterprise binaries and maintain users' access to your license.
```
Comment (ptgott) [inline docs/pages/choose-an-edition/teleport-enterprise/license.mdx:21]:
```suggestion
Teleport provides a dedicated account site where you can download your license and
enterprise binaries, and maintain users' access to your license. 
```

File: docs/pages/includes/enterprise/obtainlicense.mdx [hunk 7]
```diff
@@ -1,9 +1,9 @@
-The `teleport` binary reads a local license file to authenticate your Teleport
+The Teleport Auth Service reads a license file to authenticate your Teleport
 Enterprise account.
 
-To obtain your license file, visit the [Teleport customer
-dashboard](https://dashboard.gravitational.com/web/login) and log in. Click
-"DOWNLOAD LICENSE KEY". You will see your current Teleport Enterprise account
-permissions and the option to download your license file:
+To obtain your license file navigate to your Teleport Account and login.
```
Comment (ptgott) [inline docs/pages/includes/enterprise/obtainlicense.mdx:4]:
```suggestion
To obtain your license file navigate to your Teleport account and log in.
```
"login" is a noun

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx (hunk 5 above)
Comment (stevenGravy) [reply docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx]: updated

**Review by klizhentas — APPROVED** [review]

File: docs/pages/includes/enterprise/obtainlicense.mdx [hunk 8]
```diff
@@ -1,9 +1,9 @@
...
+You can start at [teleport.sh](https://teleport.sh) and enter your
+Teleport Account name (e.g. my-license). After logging in
+you will see the "DOWNLOAD LICENSE KEY" button to download the license file.
```
[the remark this answers is not in the document]
Comment (stevenGravy) [reply docs/pages/includes/enterprise/obtainlicense.mdx]: See update

**Comment by stevenGravy** [comment]
Your update requests should be applied @ptgott . Made small verbiage change on the obtain license one.

**Review by ptgott — APPROVED** [review]

**Pull request merged**